### ISSUE TITLE
Project Owners 101 Video: Our guide to promoting your project

### ISSUE THREAD

**Issue by ahmadabugosh** [Giveth/giveth-planning#739]

**Comment by ahmadabugosh** [comment]
I would mention

- Add a detailed description of your project 
- Give regular updates
- Mention your project to your own audience (newsletter, social media). If they're new to crypto, share our Donor course with them.
- Tag @givethio on twitter with a link to your project, what you do and we'll retweet you 
- Attend one of our weekly community calls (we need to tell them who to reach out to on discord)

**Comment by ahmadabugosh** [comment]
@SymSL to do this one

**Comment by SymSL** [comment]
Thanks @ahmadabugosh the points have been great to start the script. I've put some content together here: https://docs.google.com/document/d/13qOxq-M7_DBGnpA5uS0d_2c1Z931KsXR1trOO00szwA/edit but wanted to include points from Claires document above - its a really good guide. Is it published somewhere already that can be linked? I did a search for the heading and couldnt see it up yet. I'll add to this more tomorrow but comments in the mean time are most welcome of course

**Comment by ahmadabugosh** [comment]
@Cotabe gave a great suggestion to also mention how to use Discord as a project owner, to give themselves the "Maker" role, for them to stay up to date on ways to work with us to promote their projects.

**Comment by ahmadabugosh** [comment]
@SymSL The script looks good! Do you want to record it, or do you need support?

**Comment by SymSL** [comment]
Hey @ahmadabugosh I've done basic edits but didn't end the video at the right place. There are still some imperfections with timing so I'm happy to do another recording but thought I'd get some feedback on this one in the mean time 

https://we.tl/t-8HhvF1ybUS

**Comment by ahmadabugosh** [comment]
Thanks @SymSL , nice work! 

My initial feedback is: 

1) I think it's good but like you said, there are some timing issues in some places

2) I wouldn't mention how this is for the "Projects 101 course", since we may want to use it as a standalone video, so just say what the video is about "Helping you promote your project etc"

3) Can you record it without the logo on the bottom right and the cropping on the top and bottom? If not no worries I can crop it out. 

4) Can you show more details and examples? For instance, 
a) When you talk about how to make your project stand out, give more examples about what impact would be (how many people you helped, how many wells you dug etc.). Maybe for this one, you can also show an example of a project with a good description on the screen. 

b) When you talk about Discord, it would be nice for them to see what channel to go to and how to give themselves the "Maker role". A lot of project owners are new to Discord and would need to be shown that step by step.

c) You should mention some of the channels and things they can do on Discord, like ask questions on the support chanel, participate in AMA's etc, while showing what that looks like in Discord. 

d) When you mention the steps about Social media sharing, I would nest the channels you mention under "create your own channel" and mention that after you mention Discord for that slide. 

e) You can mention how we have a weekly community call on Thursday (and also mention how we have regular AMA calls) and explain what they are and how they can benefit from them. 

f) When you mention at the end our content, I would show our medium, docs, twitter, and also our newsletter. You can also mention that if they verify their project (which we cover in another video), we'll feature them more prominently on twitter and our newsletter. 

---
Basically just showing more of what everything you mentioned looks like. 


Also, it doesn't need to be perfect (ex. if you make a mistake, you can say it again, and I can cut out any mistakes)

Hope that helps! 🙏

**Comment by SymSL** [comment]
Thanks for the feedback Ahmad, I'm including more in the script to cover these points before recording again. I've been trying to see how to add roles in Discord, but tbh I cant see how to do it. I've gone to the assign roles channel, maybe I've missed something. I agree with mentioning Discord but will just keep it brief in this one as I didnt want this to get too long. I've made changes to incorporate most of the other points, and will record soon

**Comment by ahmadabugosh** [comment]
Hey for the discord roles, I think it's just from the ["assign-roles" channel](https://discord.com/channels/679428761438912522/902259962540077146) and choosing the "Maker" role
<img width="804" alt="Screen Shot 2022-09-19 at 11 24 19 AM" src="https://user-images.githubusercontent.com/4855928/190968196-7f22421e-7f27-4bba-958d-84f57a853d79.png">

**Comment by SymSL** [comment]
Hey @ahmadabugosh i've done another recording taking into account all the comments, here is the link with an editable video file and an exported one, hope you are able to edit them further (I did a basic edit and cut out the majority of mistakes, but there are some parts (like showing some of our social pages) that could be moved into different places, especially if you are able to separate the video and audio to place the pages where the audio mentions them. If that makes sense. https://we.tl/t-P3TUrXIedk let me know your thoughts on it and if it needs any major changes!

**Comment by SymSL** [comment]
Hey @ahmadabugosh following up on this, need me to resend this link?

**Comment by ahmadabugosh** [comment]
Thanks @SymSL, nice work! 

I edited and uploaded your video here: https://youtu.be/S6B9bf0GMpk

@WhyldWanderer can you give it a watch and let us know if it's good to go live?

**Comment by c0ric0ri** [comment]
Hey @WhyldWanderer can you review and confirm by tomorrow so if we need to correct anything we can still do that this week? Would be great to move this one to done!

**Comment by WhyldWanderer** [comment]
Looks good.. 
One suggestion: Maybe we want to link this document in the description?
https://docs.giveth.io/whatisgiveth/fundraisingGuide/

Besides that, I think we can move this one to done!
